src/styles/main.css

```
@@ -1,53 +1,66 @@
+	background: #fbbf24 !important;
+	box-shadow: 0 0 0 4px rgba(245, 158, 11, 0.16);
+	transform: translateY(-1px);
 }
 
 .sl-link-button svg {
-  width: 1.5em;
-  height: 1.5em;
-  color: var(--sl-color-white);
+	width: 1.5em;
+	height: 1.5em;
+	color: currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)